blog/2017-03-15/index.php

```diff
@@ -26,8 +26,8 @@
 require_once("../../header.php");
 ?>
 
-<link rel="stylesheet" href="<?php echo $site_root; ?>media/fancybox/jquery.fancybox-1.3.4.css" media="screen">
-<script src="<?php echo $site_root; ?>jquery.min.js"></script>
```

Comment:
@ktiedt Can't pinpoint the exact place mentioning that atm, as they kept switching website, but if you look at the current website source code and search for jquery, you'll see that it's served from a CDN ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js, even though it should come from elsewhere (OpenShift server), though, it's a Google CDN, which might cause cause issues in China, as we were talking about on Discord server, sigh xD